Test failure: TestPanicked fails with Go 1.13.1

Hi, I just cloned the latest master and tried to run the tests. I see the following failure:

```plaintext
--- FAIL: TestPanicked (0.00s)
    assert.go:101: Error Trace: 
        - app/pkg/errors/errors_test.go:91
        - ContainsSubstring assertion failed. 
         Substring: 
        		 
        - github.com/getfider/fider/app/pkg/errors_test.TestPanicked:97 (app/pkg/errors/errors.go:40)
        - Boom!
         Actual: 
        		 [string] Error Trace: 
        - goroutine 25 [running]:
        runtime/debug.Stack(0x6bb040, 0xc000097990, 0x0)
        	/usr/lib/go/src/runtime/debug/stack.go:24 +0xab
        github.com/getfider/fider/app/pkg/errors.Panicked(0x63aca0, 0x6b7930, 0x6b7930, 0xc0000182e9)
        	/home/marc/go/src/github.com/getfider/fider/app/pkg/errors/errors.go:41 +0xf5
        github.com/getfider/fider/app/pkg/errors_test.TestPanicked.func1()
        	/home/marc/go/src/github.com/getfider/fider/app/pkg/errors/errors_test.go:86 +0x69
        panic(0x63aca0, 0x6b7930)
        	/usr/lib/go/src/runtime/panic.go:679 +0x1b2
        github.com/getfider/fider/app/pkg/errors_test.TestPanicked(0xc0000ec600)
        	/home/marc/go/src/github.com/getfider/fider/app/pkg/errors/errors_test.go:97 +0x7f
        testing.tRunner(0xc0000ec600, 0x689c88)
        	/usr/lib/go/src/testing/testing.go:909 +0x19a
        created by testing.(*T).Run
        	/usr/lib/go/src/testing/testing.go:960 +0x652
         (app/pkg/errors/errors.go:41)
        - github.com/getfider/fider/app/pkg/errors_test.TestPanicked:84 (app/pkg/errors/errors.go:40)
        - Boom!
FAIL
FAIL	github.com/getfider/fider/app/pkg/errors	0.013s
```

Apparently, my stacktrace shows the place where the line where `defer` is declared (`84`) rather than the line where the panic is triggered (`97`).

My Go version is the following:

``` 
go version go1.13.1 linux/amd64
```

Changing the line number `97` -> `84` makes it pass, but of course I expect it to break runs on your side.

Can you reproduce the issue?

--- a/.circleci/config.yml
+++ b/.circleci/config.yml
@@ -14,13 +14,13 @@ jobs:
       - checkout
       - restore_cache:
           keys:
-            - fider-v1-npm-{{ checksum "package-lock.json" }}
+            - fider-v2-npm-{{ checksum "package-lock.json" }}
       - run:
           name: npm install
           command: if [ ! -d "./node_modules" ]; then npm ci; fi
       - save_cache:
           name: save node_modules cache
-          key: fider-v1-npm-{{ checksum "package-lock.json" }}
+          key: fider-v2-npm-{{ checksum "package-lock.json" }}
           paths:
             - node_modules/
 

--- a/.circleci/config.yml
+++ b/.circleci/config.yml
@@ -1,7 +1,7 @@
 version: 2
 references:
   builder: &builder 
-    getfider/circleci:0.0.20
+    getfider/circleci:0.0.21
   workdir: &workdir 
     /go/src/github.com/getfider/fider
 

--- a/.circleci/config.yml
+++ b/.circleci/config.yml
@@ -14,13 +14,13 @@ jobs:
       - checkout
       - restore_cache:
           keys:
-            - fider-v2-npm-{{ checksum "package-lock.json" }}
+            - fider-v3-npm-{{ checksum "package-lock.json" }}
       - run:
           name: npm install
           command: if [ ! -d "./node_modules" ]; then npm ci; fi
       - save_cache:
           name: save node_modules cache
-          key: fider-v2-npm-{{ checksum "package-lock.json" }}
+          key: fider-v3-npm-{{ checksum "package-lock.json" }}
           paths:
             - node_modules/
 
@@ -32,7 +32,7 @@ jobs:
       - checkout
       - restore_cache:
           keys:
-            - fider-v1-npm-{{ checksum "package-lock.json" }}
+            - fider-v3-npm-{{ checksum "package-lock.json" }}
       - run: mage lint
       - setup_remote_docker
       - run: mage build:docker
@@ -85,7 +85,7 @@ jobs:
       - checkout
       - restore_cache:
           keys:
-            - fider-v1-npm-{{ checksum "package-lock.json" }}
+            - fider-v3-npm-{{ checksum "package-lock.json" }}
       - run:
           name: test
           command: mage test:ui

--- a/CONTRIBUTING.md
+++ b/CONTRIBUTING.md
@@ -19,8 +19,8 @@ If you know these technologies or would like to learn them, lucky you! This is t
 
 | Software  | How to install | What is it used for |
 |---|---|---|
-| Go 1.11+ | https://golang.org/ | To compile server side code |
-| Node.js 10+ | https://nodejs.org/ or run `nvm use` if you have nvm installed | To compile TypeScript and bundle all the client side code |
+| Go 1.13+ | https://golang.org/ | To compile server side code |
+| Node.js 12+ | https://nodejs.org/ or run `nvm use` if you have nvm installed | To compile TypeScript and bundle all the client side code |
 | Docker | https://www.docker.com/ | To start local PostgreSQL instances |
 | cosmtrek/air | `go get github.com/cosmtrek/air/cmd/air/` | Live reload for Go apps. When you change the code, it automatically recompiles the application |
 | joho/godotenv | `go get github.com/joho/godotenv/cmd/godotenv/` | To load environment variables from a `.env` so that you don't have to change your machine environment variables |

--- a/app/pkg/errors/errors_test.go
+++ b/app/pkg/errors/errors_test.go
@@ -89,7 +89,7 @@ runtime/debug.Stack`)
 			Expect(err.Error()).ContainsSubstring(`
 github.com/getfider/fider/app/pkg/errors.Panicked`)
 			Expect(err.Error()).ContainsSubstring(`
-- github.com/getfider/fider/app/pkg/errors_test.TestPanicked:97 (app/pkg/errors/errors.go:40)
+- github.com/getfider/fider/app/pkg/errors_test.TestPanicked:84 (app/pkg/errors/errors.go:40)
 - Boom!`)
 		}
 	}()

